### DISCUSSION TITLE
Group  option

### DISCUSSION THREAD

**Discussion by RamiKrispin** [community/community#6406] [Projects and Issues]
Hello, 

Thank you for the new project's format, really like it! 

Just small feedback about the `Group` option on the tab, it seems this option is only indicated if the group option is in use. It would be nice if you can actually apply the grouping from this menu.

Thanks!

![image](https://user-images.githubusercontent.com/12760966/137566095-ac88378c-b2cf-4fb9-ba7a-84ce79da8ca9.png)

**Comment by tklein1801** [comment]
Yes, and it would be nice if you could group issues by their assignees.
